### PR TITLE
{bio}[foss/2022b] NGSpeciesID v0.3.0, medaka v1.9.1

### PR DESCRIPTION
**Pull request by akesandgren**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in total)
b-cn1603.hpc2n.umu.se - Linux Ubuntu 22.04, x86_64, AMD EPYC 7313 16-Core Processor, 1 x NVIDIA NVIDIA A100 80GB PCIe, 525.147.05, Python 3.10.12
See https://gist.github.com/akesandgren/604e4c875c0639b35ec00c4767c8622b for a full test report.

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in total)
b-cn1603.hpc2n.umu.se - Linux Ubuntu 22.04, x86_64, AMD EPYC 7313 16-Core Processor, 1 x NVIDIA NVIDIA A100 80GB PCIe, 525.147.05, Python 3.10.12
See https://gist.github.com/akesandgren/af34acd673781af0aaf7f8e9c8888def for a full test report.

File: easybuild/easyconfigs/n/NGSpeciesID/NGSpeciesID-0.3.0-foss-2022b.eb [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+name = 'NGSpeciesID'
+version = '0.3.0'
+
+homepage = 'https://github.com/ksahlin/NGSpeciesID'
+description = "NGSpeciesID is a tool for clustering and consensus forming of targeted ONT reads."
+
+toolchain = {'name': 'foss', 'version': '2022b'}
+
+sources = [SOURCE_TAR_GZ]
+
```
Comment (verdurin) [inline easybuild/easyconfigs/n/NGSpeciesID/NGSpeciesID-0.3.0-foss-2022b.eb:12]: @akesandgren - missing checksum.

**Review by verdurin — CHANGES_REQUESTED** [review]

**Review by verdurin — APPROVED** [review]
Looks fine.

**Comment by verdurin** [comment]
@boegelbot please test @ generoso

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 9 out of 9 (5 easyconfigs in total)
easybuild-c7.novalocal - Linux CentOS Linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/verdurin/6a2a33c052e68f57ea613ef7515ab394 for a full test report.

**Comment by verdurin** [comment]
@boegelbot please test @ jsc-zen3

**Comment by verdurin** [comment]
Going in, thanks @akesandgren!

**Pull request merged**